program_intent_engagement/apps/api/v1/views.py

```diff
@@ -112,12 +112,13 @@ def grab_best_intents(cls, user_intents):
         ):
             intents = list(intents)
 
-            # sort by certainty, effective_timestamp
+            # sort by certainty ascending, effective_timestamp descending
             intents.sort(
                 key=lambda x: (
-                    certainty_order[(x["certainty"])],
+                    -certainty_order[(x["certainty"])],
                     x["effective_timestamp"],
-                )
+                ),
+                reverse=True
```

Comment:
you need both this statement and the `-` sign?

Reply:
Yes, both would be needed for this to work @schenedx . But I'm looking into having multiple sorts chained as @ashultz0 suggested in the chat so hopefully can avoid this.